rocAL/rocAL/source/decoders/image/fused_crop_decoder.cpp

```diff
@@ -138,40 +138,38 @@ Decoder::Status FusedCropTJDecoder::decode(unsigned char *input_buffer, size_t i
             y1 =  (original_image_height - crop_height) / 2;
         }
     }
-    
-   // std::cout<<"Fused Crop Decoder <x,y, w, h>: " << x1 << " " << y1 << " " << crop_width << " " << crop_height << std::endl;
+    if(crop_width > max_decoded_width)
+        crop_width = max_decoded_width;
+    if(crop_height > max_decoded_height)
+        crop_height = max_decoded_height;
```

Comment:
Add a comment why we are handling this case.

Reply:
Done